src/python/nimbusml/examples/NGramExtractor.py

```diff
@@ -0,0 +1,39 @@
+###############################################################################
+# NGramFeaturizer
+from nimbusml import FileDataStream, Pipeline
+from nimbusml.datasets import get_dataset
+from nimbusml.preprocessing.schema import ColumnDropper
+from nimbusml.preprocessing.text import CharTokenizer
+from nimbusml.feature_extraction.text import NGramExtractor
+from nimbusml.feature_extraction.text.extractor import Ngram
+
+# data input (as a FileDataStream)
+path = get_dataset("wiki_detox_train").as_filepath()
+
+data = FileDataStream.read_csv(path, sep='\t')
+print(data.head())
+#   Sentiment                                      SentimentText
+# 0          1  ==RUDE== Dude, you are rude upload that carl p...
+# 1          1  == OK! ==  IM GOING TO VANDALIZE WILD ONES WIK...
+# 2          1  Stop trolling, zapatancas, calling me a liar m...
+# 3          1  ==You're cool==  You seem like a really cool g...
+# 4          1  ::::: Why are you threatening me? I'm not bein...
+
+# transform usage
+pipe = Pipeline([
+        CharTokenizer(columns={'SentimentText_Transform': 'SentimentText'}),
+        NGramExtractor(ngram_length=1, all_lengths=False, columns={'Ngrams': 'SentimentText_Transform'}),
+        ColumnDropper(columns=['SentimentText_Transform', 'SentimentText', 'Sentiment'])
```

Comment:
I think I'm missing something here. How does `Sentiment` get dropped, then printed below? Are we looking to teach users to explicitly drop no longer needed cols like `SentimentText`?